plugins/shopify/src/editors/ShopifyProductPicker.tsx

```diff
@@ -94,9 +94,9 @@ export class ProductPicker extends SafeComponent<
     // const agent =
     // TODO: cancen pending requests if any
     const productsResponse = await fetch(
-      `${shopifyProductsUrl}?apiKey=${this.props.context.user.apiKey}&title=${encodeURIComponent(
-        this.searchInputText
-      )}&limit=40`
+      `${shopifyProductsUrl}?apiKey=${this.props.context.user.apiKey}&limit=40${
+        this.searchInputText ? `&search=title:*${this.searchInputText}*` : ''
```

Comment:
```suggestion
        this.searchInputText ? '&search=' + encodeURIComponent(`title:*${this.searchInputText}*`) : ''
```
or maybe
```suggestion
        this.searchInputText ? `&search=title:*${encodeURIComponent(this.searchInputText)}*` : ''
```
?

Reply:
neither sent the expected results

Reply:
updated to use storefront client instead, it's better this way so only product availble for the private app show up